components/admin/Courses/Courses.jsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react'
+import { Disclosure } from '@headlessui/react'
+import { ChevronUpIcon } from '@heroicons/react/solid'
+import { CourseTable } from './CourseTable'
+
+export const Courses = () => {
+  return (
+    <>
+    <div className="w-full px-4 pt-16">
```

Comment:
headlessui disclosures to allow admin to choose which course they want to look at